clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -78,35 +78,35 @@ public long connectionDelay(String id, long now) {
 
     /**
      * Enter the connecting state for the given connection.
-     * @param id The id of the connection
+     * @param id The id of the connection.
      * @param now The current time.
      */
     public void connecting(String id, long now) {
         nodeState.put(id, new NodeConnectionState(ConnectionState.CONNECTING, now));
     }
 
     /**
-     * Return true iff a specific connection is connected
-     * @param id The id of the connection to check
+     * Return true if a specific connection is connected.
```

Comment:
"iff" means "if and only if" and is correct in this context. I don't think it was a typo.

If you find "iff" unclear, feel free to replace with "if and only if"